requirements.txt

```diff
@@ -2,3 +2,5 @@ beautifulsoup4==4.5.3
 PyMuPDF==1.17.5
 selenium==3.141.0
 urllib3==1.25.10
+spacy==2.3.2
```

Comment:
@joel-lim Do we still need this?

Reply:
My bad, I forgot to commit the new requirements.txt